### DISCUSSION TITLE
Exclude internal local IP from the stats

### DISCUSSION THREAD

**Discussion by wrtz008** [YOURLS/YOURLS#3824] [Q&A] [answered]
Is there any way to exclude specific IPs like my local IP from the Stats showing in Yourl both for IP column and Clicks column?

Thank you!
![Yourls](https://github.com/user-attachments/assets/edbb1bd8-36f7-4125-98c8-38121ff00dc7)

**Comment by dgw** [comment]
Couldn't find anything exactly like this, but it shouldn't be too difficult to adapt another "skip tracking" plugin like [mine](https://github.com/dgw/yourls-dont-track-admins) to trigger on the IP (fetch using `yourls_get_IP()`) or subnet ([little more complicated](https://stackoverflow.com/a/594134/5991), but can be useful if your IP is dynamic).

Making it configurable through the admin area, without editing the plugin file code, would be much more work. But the basic functionality seems simple enough.

**Reply by wrtz008** [reply]
thanks a lot @dgw !!! wow! i did not know about your plugin!

I've installed but how do I use the `yourls_get_IP()`? Do I add this line somewhere in the main Yourls plugin?

Thanks!

**Reply by dgw** [reply, marked as answer]
Installing my plugin isn't going to do what you want. You have to create your own plugin called something like "Exclude Local IP", which will use the `yourls_get_IP()` global function to get the user's IP and compare it to one (or a list) that you set as excluded.

Not something I can dash off as an example today, nor would I want to just dump untested code on someone. If you aren't comfortable enough with PHP programming to make this yourself, you'll need help.

**Reply by wrtz008** [reply]
thx @dgw, got it, appreciated! understood!
